logging/src/logging/truncator.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+    Copyright (c) 2018 Artem Selishchev <artsel@yandex-team.ru>
+    Copyright (c) 2018 Other contributors as noted in the AUTHORS file.
+
+    This file is part of Cocaine.
+
+    Cocaine is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    Cocaine is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "cocaine/logging/truncator.hpp"
+
+namespace cocaine {
+namespace logging {
+namespace defaults {
+
+constexpr auto string_size_limit = std::numeric_limits<std::uint64_t>::max();
```

Comment:
It could be `size_t` as it it type for string size, but by definition:

> std::string::npos
> is a static member constant value with the greatest possible value for an element of type size_t.

Reply:
Changed to `std::numeric_limits<std::size_t>::max()`